LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2019 Cybozu
+Copyright (c) 2020 The TopoLVM Authors
```

Comment:
This copyright started in 2019. So 2019 would be appropriate here.

Reply:
Ah, I misunderstood that "Copyright (c) 2020" replaces "Copyright (c) 2019". This change looks good to me. Sorry for the noise.

Reply:
Could you approve this?